packetbeat/protos/dns/dns.go

```diff
@@ -469,7 +469,7 @@ func addDnsToMapStr(m common.MapStr, dns *mkdns.Msg, authority bool, additional
 
 	m["answers_count"] = len(dns.Answer)
 	if len(dns.Answer) > 0 {
-		m["answers"] = rrsToMapStrs(dns.Answer)
+		m["answers"] = ansToMapStrs(dns.Answer)
```

Comment:
If we went this route, then we would need to apply the same transformation to the `dns.authorities` field.